physionet-django/events/views.py

```diff
@@ -22,9 +22,16 @@
 
 @login_required
 def update_event(request, event_slug, **kwargs):
+
     user = request.user
     can_change_event = user.has_perm('events.add_event')
     event = Event.objects.get(slug=event_slug)
+
+    # if the event has dataset added to it, it cannot be edited
```

Comment:
This is not intuitive behaviour, and doesn't (as far as I can see) relate to the original goal of the PR.

Reply:
This was a feature requested by Benjamin upon discussion.